app/views/work_packages/_two_column_attributes.html.erb

```diff
@@ -29,12 +29,12 @@ See doc/COPYRIGHT.rdoc for more details.
 <% left_attributes, right_attributes = attributes.in_groups(2, false) %>
 
 <div class="grid-block wrap">
-  <div class="form--column">
+  <div class="form--column medium-6">
     <% left_attributes.each do |attribute| %>
       <%= attribute.field %>
     <% end %>
   </div>
-  <div class="form--column">
+  <div class="form--column medium-6">
```

Comment:
That should happen by itself. Why do you need to specify it? Additionally, if something like this is necessary, we should add those classes as subclasses of `form--column` and not rely on foundation directly.